README.md

```diff
@@ -317,6 +321,42 @@ The batch size to use when bulk creating `Messages` in the database. The default
 
 The time in seconds to keep `Messages` in the database before deleting them. `None` means the messages will be kept indefinitely, `0` means no messages will be kept, and any other integer value will be the number of seconds to keep messages. The default is `None`.
 
+#### `RELAY_HEALTHCHECK_METHOD`
+
+| Component     | Configurable |
+|---------------|--------------|
+| Relay Service | Yes ✅       |
+| Django App    | No 🚫        |
+
+The HTTP method to use for the healthcheck endpoint. [`RELAY_HEALTHCHECK_URL`](#relay_healthcheck_url) must also be set for this to have any effect. The default is `"GET"`.
```

Comment:
I'm not entirely sure what this should be set to. I know the relay is running, but I wasn't entirely sure if it'd be be exposed to the web.

Reply:
Yeah, at least in our environment the relay is in the DMZ, so it has access to the SMTP server internally and database in Crunchy. But it shouldn't need to be exposed to the web, just have access to the web. It's a healthcheck ping after the loop runs. In our case we use Healthchecks.io for any cronjobs.

Maybe a different name would be better? Heartbeat?